RibonDAO/core-api
# Feat/new days donating

Adds a new field to user donation stats, and a new job to increment the days donating field on each donation.

It also adds a one time run job, that should update the new field for all users in batch

## Type of change


## How to test

- Run the IncrementDaysDonatingJob command
  - Check if it only increments days donating if there wasnt a donation on the same day
- Run the UpdateDaysDonatingForAllJob 
  - Check if the job updates the value correctly for all users
  - Also check if in case of failure, the job continues from where it stopped
